docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -17,25 +17,41 @@ default:: `syn-minio`
 type:: string
 default:: `admin-credentials-${_instance}`
 
-== `adminCredentials.acessKey`
+== `adminCredentials.accessKey`
 
 [horizontal]
 type:: string
 default:: Vault reference
 
-A vault reference pointing to the Vault secret containing the access key.
-If the key isn't provided as a vault reference, the secret key must be provided in its base64 encoded form.
-It can be generated using the command `echo -n '<myAccessKey> | base64`.
+A vault reference pointing to the Vault secret containing the rootUser or a string with the username for the admin user
+
+NOTE: Deprecated: Use `adminCredentials.rootUser` parameter instead.
 
 == `adminCredentials.secretKey`
 
 [horizontal]
 type:: string
 default:: Vault reference
 
-A vault reference pointing to the Vault secret containing the access key.
-If the key isn't provided as a vault reference, the secret key must be provided in its base64 encoded form.
-It can be generated using the command `echo -n '<mySecretKey> | base64`.
+A vault reference pointing to the Vault secret containing the rootUser or a string with the username for the admin user
+
+NOTE: Deprecated: Use `adminCredentials.rootPassword` parameter instead.
+
```

Comment:
Personal taste: I would completely remove the documentation for the deprecated parameters

Reply:
yeah makes sense. Removed them